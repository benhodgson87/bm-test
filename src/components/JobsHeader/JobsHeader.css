.jobs-header {
  display: flex;
  align-items: center;
  padding: 10px;
  margin-bottom: 4px;
  background-color: #fff;
}

.jobs-header h2 {
  margin: 0;
  font-size: 16px;
  color: #2673b4;
}
