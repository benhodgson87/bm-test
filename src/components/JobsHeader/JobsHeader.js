import React from 'react'
import { Row, Col } from 'react-flexbox-grid'
import JobSorting from '../JobSorting'

import './JobsHeader.css'

const JobsHeader = () => (
  <Row className="jobs-header">
    <Col xs={8}>
      <h2>Tasks</h2>
    </Col>
    <Col xs={2}>{/* TODO: To Cube - not sure what this is, check with BM devs */}</Col>
    <Col xs={2}>
      <JobSorting />
    </Col>
  </Row>
)

export default JobsHeader
