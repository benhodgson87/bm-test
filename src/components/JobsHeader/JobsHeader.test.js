import React from 'react'
import { shallow } from 'enzyme'

import JobsHeader from './JobsHeader'

describe('JobsHeader component', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<JobsHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
