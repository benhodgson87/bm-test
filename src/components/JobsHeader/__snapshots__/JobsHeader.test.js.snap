// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobsHeader component renders correctly 1`] = `
<Row
  className="jobs-header"
>
  <Col
    xs={8}
  >
    <h2>
      Tasks
    </h2>
  </Col>
  <Col
    xs={2}
  />
  <Col
    xs={2}
  >
    <Connect(JobSortingContainer) />
  </Col>
</Row>
`;
